cfme/tests/v2v/test_rest_migrations.py

```diff
@@ -0,0 +1,183 @@
+"""Test to check v2v migration rest API"""
+import fauxfactory
+import pytest
+
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_VERSION, ONE_PER_TYPE
+
+
+pytestmark = [
+    pytest.mark.provider(
+        classes=[RHEVMProvider],
+        selector=ONE_PER_VERSION,
+        required_flags=["v2v"]
+    ),
+    pytest.mark.provider(
+        classes=[VMwareProvider],
+        selector=ONE_PER_TYPE,
+        fixture_name="second_provider",
+        required_flags=["v2v"]
+    )
+]
+
+
+@pytest.fixture(scope="function")
+def get_clusters(appliance, v2v_providers):
```

Comment:
These fixtures are repeated in many of the tests, with slightly different implementations. Some tests may need slightly different sources, but I think that the v2v team needs to evaluate creating a few common cluster/network/datastore fixtures in `cfme/fixtures/v2v.py` for use in all the tests.

Reply:
@mshriver  good point here, if we need to implement this then we need to rethink + redesign it from scratch since source and target entity is first point of implementation. So this can be technically possible but .. may be I will put this in WIP and work on this and try to minimize structure and lets see hows it look.

cc @sshveta  @kedark3  I am on it, thoughts?